packages/model-registry/upstream/frontend/src/app/pages/modelRegistry/screens/ModelVersions/ModelVersionsTabs.tsx

```diff
@@ -27,6 +30,64 @@ const ModelVersionsTabs: React.FC<ModelVersionsTabProps> = ({
   mvRefresh,
 }) => {
   const navigate = useNavigate();
+  const tabExtensions = useExtensions(isModelRegistryDetailsTabExtension);
+  const { registeredModelId: rmId } = useParams();
+  const { preferredModelRegistry } = React.useContext(ModelRegistrySelectorContext);
+  
+  const modelDetailsTabs = [
+    <Tab
+      key={ModelVersionsTab.OVERVIEW}
+      eventKey={ModelVersionsTab.OVERVIEW}
```

Comment:
This is another large diff, but based on @rsun19 's previous PRs around tab extensions I guess there's not much we can do about it for now. It would be nice to eventually make this same refactor in kubeflow (just lift the existing tabs out into an array) so that this midstream diff is just adding an array element.